### PR TITLE
chore: improve app layout with auto reties and manual retry buttons

### PR DESCRIPTION
**Pull request by BlankParticle**
## What does this PR do?

The app will now automatically retry if the org layout fails to load and will give users a button to reload the page

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Required

- [ ] Read [Contributing Guide](https://github.com/un/inbox/blob/main/CONTRIBUTING.md)
- [ ] Self-reviewed my own code
- [ ] Tested my code in a local environment
- [ ] Commented on my code in hard-to-understand areas
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the UnInbox Docs if changes were necessary

### CODE REVIEW

**Comment by BlankParticle** [comment]
* **#613** <a href="https://app.graphite.dev/github/pr/un/inbox/613?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @BlankParticle and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/web/src/app/[orgShortcode]/layout.tsx [hunk 1]
```diff
@@ -29,28 +32,36 @@ export default function Layout({
     return (
       <div className="flex h-full w-full flex-col items-center justify-center gap-2">
         <SpinnerGap className="text-base-11 h-20 w-20 animate-spin" />
+        <span className="text-slate-11 font-bold">UnInbox is Loading...</span>
```
Comment (McPizza0) [inline apps/web/src/app/[orgShortcode]/layout.tsx:35]: we should always wrap `UnInbox` in a span with `font-display` so it shows our logo
can you try this and send a screenshot for confirmation please

**Review by McPizza0 — CHANGES_REQUESTED** [review]
`UnInbox`  font change

**Comment by BlankParticle** [comment]
image

**Review by McPizza0 — APPROVED** [review]

**Comment by McPizza0** [comment]
### Merge activity

* **Jul 27, 2:33 PM EDT**: @McPizza0 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/un/inbox/613).

**Pull request merged**